libraries/AC_PrecLand/AC_PrecLand.cpp

```diff
@@ -52,8 +52,8 @@ const AP_Param::GroupInfo AC_PrecLand::var_info[] = {
 
     // @Param: EST_TYPE
     // @DisplayName: Precision Land Estimator Type
-    // @Description: Specifies the estimation method used.
-    // @Values: 0:RAW_MEASUREMENTS, 1:TWO_STATE_KF_PER_AXIS
+    // @Description: Precision Land Estimator Type
```

Comment:
Was this on purpose? A description different from display name looks better, maybe a small explanation about the two methods would be good.